### ISSUE TITLE
porting from old to new version problem

### ISSUE THREAD

**Issue by pzabawa-poczta-onet-pl** [jbezos/enumitem#12]
The following line:
\begin{description}[align=left,labelwidth=\widthof{\bf\cdmmformal{IndirectConnection}}]
submits an error after upgarde of both enumitem and calc packages.
Maybe not an issue but just the same effect should be obtained in a different way.
What should I do?

**Comment by FrankMittelbach** [comment]
@pzabawa-poczta-onet-pl  is it the case that you load calc  *after* enumitem? 3.7 introduced a dependency and requires calc to be loaded first (should be fixed)

**Comment by pzabawa-poczta-onet-pl** [comment]
Hi Frank,
do you know why the problem of dependency management is not solved in LaTeX like, say in Maven? This is a common problem for LaTeX users.
Cheers
Piotr

Dnia 31 stycznia 2019 09:58:18 CET, Frank Mittelbach <notifications@github.com> napisał(a):
>@pzabawa-poczta-onet-pl  is it the case that you load calc  *after*
>enumitem? 3.7 introduced a dependency and requires calc to be loaded
>first (should be fixed)
>
>-- 
>You are receiving this because you were mentioned.
>Reply to this email directly or view it on GitHub:
>https://github.com/jbezos/enumitem/issues/12#issuecomment-459266939

--
Wysłane z aplikacji OnetPoczta

**Comment by FrankMittelbach** [comment]
> do you know why the problem of dependency management is not solved in 
> LaTeX like, say in Maven? This is a common problem for LaTeX users.

I quite understand that this is a problem for LaTeX users but I don't 
see how that could be easily solved (or how Maven manages that).

Point is the dependency is an order dependency and that is implicit due 
to using \let in the new code where \def should have been used (see 
#14). That dependency shouldn't be there in the first place, it is not 
something that could be automatically detected, nor could it have been 
"declared" as I guess Javier simply overlooked it.

Basically a dependency management for LaTeX would require a manual setup 
and management of the underlying graph which is imho not something that 
could be reliably achieved giving available resources.

**Comment by jbezos** [comment]
Thanks. I'll fix it in a few days.
